@here/olp-sdk-dataservice-read/lib/layer/LayerProvider.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { ConfigApi } from "@here/olp-sdk-dataservice-api";
+import { CatalogClient, CatalogLayer, CatalogRequest, CatalogVersionRequest, HRN, OlpClientSettings } from "..";
+
+export class LayerProvider {
```

Comment:
Please rename the class to the LayerDataProvider

Reply:
Done